packages/garbo/src/lib.ts

```diff
@@ -1054,3 +1054,37 @@ export function aprilFoolsRufus() {
     visitUrl("questlog.php?which=7");
   }
 }
+
+type LuckyAdventure = {
```

Comment:
this should contain an optional outfit and macro to be able to handle lucky adventures that are combats (mirror), as well as optional choices because a few also have choice adventures